ray-operator/controllers/ray/common/rbac.go

```diff
@@ -42,7 +42,12 @@ func BuildRole(cluster *rayv1.RayCluster) (*rbacv1.Role, error) {
 			{
```

Comment:
Does the KubeRay operator need to have `pod/resize` permission itself if it creates a Role with that permission, or is permission to create Roles sufficient?

Reply:
It needs to have the permission itself in order to create a role with that permission.